sound/soc/intel/boards/Kconfig

```diff
@@ -313,14 +312,14 @@ config SND_SOC_INTEL_BXT_DA7219_MAX98357A_COMMON
 if SND_SOC_INTEL_APL
 
 config SND_SOC_INTEL_BXT_DA7219_MAX98357A_MACH
-	tristate "Broxton with DA7219 and MAX98357A/MAX98390 in I2S Mode"
+	tristate "Broxton with DA7219 and MAX98357A in I2S Mode"
```

Comment:
same problem as last time, if a user does a 'make olddefconfig' then they may not enable the 'new' config and their sound support would be broken. For backwards compatibility using 'imply' seems required, no?

Reply:
Sorry I forgot this. Added to SND_SOC_INTEL_CML_LP_DA7219_MAX98357A_MACH config.